docs/DeveloperGuide.md

```diff
@@ -660,24 +660,134 @@ testers are expected to do more *exploratory* testing.
 
     1. Click the close button on the window.<br>
        Expected: The app closes.
-
-   2. Use the `exit` command.<br>
+       
+    2. Use the `exit` command.<br>
        Expected: The app closes.
 
-#### Deleting a person
+### Editing an internship
+1. Editing a single field of an internship
+
+1. Test case: `edit 1 /com Facebook`<br>
+   Expected: The name of the first internship is changed to `Facebook`. Details of the edited internship shown in the status message.
+
+
+2. Repeat for all other fields of an internship other than `TaskList`.
+   Expected: Similar to previous.
+
+
+3. Test case: `edit 0 n/editedName`<br>
```

Comment:
Is the test case supposed to be `edit 0 /com editedName`?
Are you targeting a prefix error or an index error